src/messages/javascript.js

```diff
@@ -0,0 +1,15 @@
+import { gettext } from 'utils';
+
+
+// These constants aren't fullcaps so we can use filenames/rule names/
+// message names consistently across the app.
+export const mozIndexedDB = {
+  code: 'mozIndexedDB',
+  message: gettext('mozIndexedDB has been removed; use indexedDB instead.'),
+  description: gettext('mozIndexedDB has been removed; use indexedDB instead.'),
+};
+export const mozIndexedDB_prop = {
+  code: 'mozIndexedDB_prop',
```

Comment:
This refers to a specific JS api so I see why it's this way but I was envisaging keeping all the codes + constants uppercase.